tests/cases/parser/namespaceUseDeclaration30.php

```diff
@@ -0,0 +1,3 @@
+<?php
+// TODO error
```

Comment:
Looks like a skipped token is already getting properly produced here, so no need for this comment.